static/js/components/About.jsx

```diff
@@ -79,40 +114,63 @@ BibLink.propTypes = {
 };
 
 const About = () => {
+  // {% block componentConsts -%}
   const classes = useStyles();
   const version = useSelector((state) => state.sysInfo.version);
   const cosmology = useSelector((state) => state.sysInfo.cosmology);
   const cosmoref = useSelector((state) => state.sysInfo.cosmoref);
   const gitlog = useSelector((state) => state.sysInfo.gitlog);
+  // {% endblock -%}
+
+  // {% block configVariables -%}
+  // Replace these with Jinja double-brace injection when extending if you just
+  // want to use a config file variable instead of this default appTitle without
+  // overwriting a render block that uses it below.
+  //
+  // This is done like this instead of just directly writing something like
+  // {{ app.title }} below because the actual double-brace injection syntax will
+  // not be valid JSX if not extending the About component.
+  const appTitle = "SkyPortal";
```

Comment:
appTitle should be important from the config.yaml, not hardcoded here.

Reply:
As I explain in the inline comments here, I don't think there's a way to import the values from the config.yaml without restructuring this PR back to the way I had it before @stefanv's suggestions. I'd have to either make this a .template file or make it an invalid .jsx file and get our linters to ignore it. I think converting it to an `About.jsx.template` is bad because the whole point of the PR was so that Fritz wouldn't have to take an entire file and rewrite it; that would lead us back to the `BaseAbout.jsx.template` that was rejected earlier in this PR. I thought this solution was an acceptable compromise. Note that while in the base SkyPortal version of the About component this is hard-coded, any application extending the page (i.e. Fritz) can import relevant app-specific configs by overwriting the `configVariables` Jinja block when extending this file.

Reply:
Simply making it possible to override the title should be fine, given that any overrides will happen in a template.  In *that* template, the title *will* be accessible.